lib/queue/worker.js

```diff
@@ -249,7 +249,7 @@ Worker.prototype.zpop = function( key, fn ) {
     .zremrangebyrank(key, 0, 0)
     .exec(function( err, res ) {
       if( err ) return fn(err);
-      var id = res[ 0 ][ 0 ];
+      var id = res[ 0 ][ 0 ] || res[ 0 ][ 1 ][ 0 ];
```

Comment:
seems `ioredis` multi/exec reply format is different than node_redis, 
Then all `.exec` calles which are using replies should be refactored in Kue... no?